docs/get_started.md

```diff
@@ -96,16 +96,20 @@ the creation of a new management pack project.
 
 For complete documentation of the `mp-init` tool including an overview of its output, see the [MP Initialization Tool Documentation](references/mp-init.md).
 
-### Template Project
+### Template Project and Entry Point
 Every new project creates a file system that has the basic project structure required to develop and build a Management Pack.
 Each file and directory is discussed in depth in the [mp-init](references/mp-init.md) documentation. `app/adapter.py` is the adapter's
-entry point and the best starting point.
+entry point and the best starting point. Users can pick between two entry points when creating their project:
+- `Template Adapter Entry Point`:  Ideal for first-time users;
+  it comes with a template adapter that collects several objects and metrics from the container the adapter is running.
+  The template adapter has comments throughout its code
+  that explain what the code does and how to customize it for your adapter.
 
-`adapter.py` is a template adapter that collects several objects and metrics from the
-container in which the adapter is running. The template adapter has comments throughout its code that explain what the code does
-and how to customize it for your adapter.
+- `Boilerplate Entrypoint`: Ideal for experienced users;
+  it comes without any examples, just the necessary code and comments to implement test connection,
+  collection, adapter definition, and endpoints logic.
 
-The methods inside the adapter template are required, and should be modified to generate a custom
+The methods inside the `adapter.py` are required, and should be modified to generate a custom
 adapter. Each method fulfills a request from the VMware Aria Operations collector, and can be tested individually using
```

Comment:
* The entry point isn't changing - it's `adapter.py` in both cases, so I think most of the references to entry point are not needed.
* I think both options could reasonably be called 'templates', so let's not use that for distinguishing between the two.

Given the above, how about something like `Sample Adapter` or `Example Adapter` for the psutil adapter. For the 'blank' adapter here are some suggestions - we can talk about pros and cons offline with Mark:
* `Blank Adapter`
* `Default Adapter`
* `Stub Adapter`
* `New Adapter`
*  for the other. I don't think [Boilerplate](https://en.wikipedia.org/wiki/Boilerplate_code#:~:text=In%20computer%20programming%2C%20boilerplate%20code,to%20accomplish%20only%20minor%20functionality.) fits here.

Reply:
I prefer `New Adapter`